src/lolex-src.js

```diff
@@ -394,6 +398,12 @@ function hijackMethod(target, method, clock) {
     target[method].clock = clock;
 }
 
+exports.shouldAdvanceTime = false;
```

Comment:
I'm also not super in-love with this :rofl: 
is this what you mean?

```
exports.config = {
    shouldAdvanceTime: false
}
```
and then toggle it through:
```
lolex.config = {shouldAdvanceTime: true};
var clock = lolex.install(date);
```

I didn't want to overload the constructor with another parameter since I didn't dive too deep into what would be the implications of rewriting the logic doing the parameter swapping magic and what would be the possible repercussions for people using this library on contexts different than sinon. Also if adding another parameter to the install function (obviously at the end of the param list - `install(target, now, toFake, loopLimit, shouldAdvanceTime)`) - now install would be invoked with `install(date,null,null,null,true)`? not sure if it makes much sense either, and not really readable at first/second/tenth sight